packages/ra-ui-materialui/src/form/SimpleForm.js

```diff
@@ -63,6 +63,27 @@ const SimpleForm = ({ initialValues, defaultValue, saving, ...props }) => {
     );
 };
 
+SimpleForm.propTypes = {
+    basePath: PropTypes.string,
+    children: PropTypes.node,
+    className: PropTypes.string,
+    defaultValue: PropTypes.oneOfType([PropTypes.object, PropTypes.func]), // @deprecated
+    initialValues: PropTypes.oneOfType([PropTypes.object, PropTypes.func]),
+    handleSubmit: PropTypes.func, // passed by react-final-form
+    invalid: PropTypes.bool,
```

Comment:
`invalid` and `pristine` are react-final-form properties that come into existence *inside* `SiumpleForm`, so they should not be PropTypes

Reply:
Done.

Reply:
Corrected.